docs/resources/Guild_Scheduled_Event.md

```diff
@@ -23,7 +23,7 @@ A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
 | entity_metadata **    | ?[entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata)   | additional metadata for the guild scheduled event                                                                                                                                                                     |
 | creator?              | [user](#DOCS_RESOURCES_USER/user-object) object                                                                                | the user that created the scheduled event                                                                                                                                                                             |
 | user_count?           | integer                                                                                                                        | the number of users subscribed to the scheduled event                                                                                                                                                                 |
-| image                 | ?string                                                                                                                        | the [cover image hash](#DOCS_REFERENCE/image-formatting) of the scheduled event                                                                                                                                       |
+| image?                | ?string                                                                                                                        | the [cover image hash](#DOCS_REFERENCE/image-formatting) of the scheduled event                                                                                                                                       |
```

Comment:
this affected a library https://github.com/Rapptz/discord.py/issues/7501 :laughing: